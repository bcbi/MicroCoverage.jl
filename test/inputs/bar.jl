function bar(arr)
    if "aaa" in arr && "zzz" in arr
        return "hello"
    else
        return "goodbye"
    end
end
