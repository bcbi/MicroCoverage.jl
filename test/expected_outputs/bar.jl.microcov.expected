[1,1,1]                 function bar(arr)
[1,1,0,1,1]                 if "aaa" in arr && "zzz" in arr
[0]                             return "hello"
-                           else
[1]                             return "goodbye"
-                           end
-                       end
-                       
